src/apps/index.ts

```diff
@@ -24,12 +25,12 @@ export async function index(): Promise<Array<App>> {
     throw "Could not find your local UCAN"
   }
 
-  const jwt = await ucan.build({
+  const jwt = ucan.encode(await ucan.build({
+    attenuations: [],
     audience: await api.did(),
     issuer: await did.ucan(),
-    proof: ucan.encode(localUcan), 
-    potency: null
-  })
+    proof: ucan.encode(localUcan)
```

Comment:
Proofs are also an array in 0.4. Not sure if `ucan.encode` handles that yet, but I'm guessing no.

Reply:
Oh thanks, I missed that.